modxcms/revolution
# Fix tab tracking issue for plugin panel

### What does it do?
Adds a check for the presence of a `bottomToolbar` before attempting to `changePage` (in the global grid methods `applyGridFilter` and `clearGridFilters`).

### Why is it needed?
When tab panels contain a mix of grid and non-grid tabs, changing between tabs would cause an error due to the attempt to run a grid method (`changePage`) on a tab not containing a grid.

### How to test
Clear caches and click on a plugin to edit. Verify that changing tabs works as expected.

### Related issue(s)/PR(s)
Resolves #16362
